Common/include/CConfig.hpp

```diff
@@ -5064,11 +5073,17 @@ class CConfig {
   const su2double *GetWall_Catalycity(void) const { return Wall_Catalycity; }
 
   /*!
-   * \brief Provides the gas mass fractions of the flow
-   * \return: Gas Mass fractions
+   * \brief Provides the gas mass fractions of the flow.
+   * \return: Gas Mass fractions.
```

Comment:
This is a bit nitpicky. but I dont think ":" should be there.

Reply:
yes please remove, IDK how picky doxygen is, but \return it the canonical form

Reply:
After looking through the code, most briefs have a period, so Ill go ahead and resolve this.